README.md

```diff
@@ -1,154 +1,139 @@
-# Nerves.Runtime
+# Nerves Runtime
```

Comment:
I'm not a big fan of introducing another way of referring to projects. This is a general Elixir comment, since Elixir forces two names on you automatically (i.e., the OTP application name and the main module name). I think that we should pick one and go with it.